portfolio/src/main/webapp/index.html

```diff
@@ -108,6 +108,16 @@ <h4>Fun Fact Roulette</h4>
           </a>
           <span id="fact-container"></span>
         </div>
+        <h4>Leave me a comment: </h4>
+        <form action="/data" method="POST">
+          <label for="username">Name: </label>
```

Comment:
I think the for attribute is supposed to match the id of another component, so it knows what it's labelling.